src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -14,6 +18,39 @@ class Tribe__Tickets__Repositories__Attendee__Commerce extends Tribe__Tickets__A
 	 */
 	protected $key_name = 'tribe-commerce';
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->attendee_provider = tribe( 'tickets.commerce.paypal' );
+
+		$this->create_args['post_type'] = $this->attendee_provider->attendee_object;
+
+		// Use a regular variable so we can get constants from it PHP <7.0
+		$attendee_provider = $this->attendee_provider;
+
+		// Add object specific aliases.
+		$this->update_fields_aliases = array_merge( $this->update_fields_aliases, [
+			'ticket_id'       => $attendee_provider::ATTENDEE_PRODUCT_KEY,
+			'event_id'        => $attendee_provider::ATTENDEE_EVENT_KEY,
+			'post_id'         => $attendee_provider::ATTENDEE_EVENT_KEY,
+			'security_code'   => $attendee_provider->security_code,
+			'order_id'        => $attendee_provider->order_key,
+			'optout'          => $attendee_provider->attendee_optout_key,
+			'user_id'         => $attendee_provider->attendee_user_id,
+			'price_paid'      => '_paid_price',
+			'price_currency'  => '_price_currency_symbol',
+			'full_name'       => $attendee_provider->full_name,
+			'email'           => $attendee_provider->email,
+			'attendee_status' => $attendee_provider->attendee_tpp_key,
+			'refund_order_id' => $attendee_provider->refund_order_key,
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 45251e5f4da0d4d8b4391a2a077552da</sub>